core/modules/qdisp/ResponseRequester.h

```diff
@@ -20,8 +20,8 @@
  * the GNU General Public License along with this program.  If not,
  * see <http://www.lsstcorp.org/LegalNotices/>.
  */
-#ifndef LSST_QSERV_QDISP_RESPONSEREQUESTER_H
-#define LSST_QSERV_QDISP_RESPONSEREQUESTER_H
+#ifndef LSST_QSERV_QDISP_RESPONSEHANDLER_H
+#define LSST_QSERV_QDISP_RESPONSEHANDLER_H
```

Comment:
This should be done after renaming the file, but probably OK if you are going to rename it soon.